WebApiCRUD/appsettings.json

```diff
@@ -0,0 +1,16 @@
+{
+  "Logging": {
+    "LogLevel": {
+      "Default": "Information",
+      "Microsoft.AspNetCore": "Warning"
+    }
+  },
+  "AllowedHosts": "*",
+  "ConnectionStrings": {
+    "DefaultConnection": "Server=.;Database=CRUD;TrustServerCertificate=True;Integrated Security=true; MultipleActiveResultSets=true"
```

Comment:
See if you can implement "Options" design pattern for DB ConnectionString and JWT key.